packages/core/src/editor/model/Editor.ts

```diff
@@ -1136,4 +1137,69 @@ export default class EditorModel extends Model {
       el[varName][name] = value;
     }
   }
+
+  private sendTelemetryData() {
+    const hostName = window.location.hostname;
+
+    // @ts-ignore
+    const enableDevTelemetry = __ENABLE_TELEMETRY_LOCALHOST__;
+
+    if (!enableDevTelemetry && (hostName === 'localhost' || hostName.includes('localhost'))) {
+      // Don't send telemetry data for localhost
+      return;
+    }
+
+    const sessionKeyPrefix = 'gjs_telemetry_sent_';
+
+    // @ts-ignore
+    const version = __GJS_VERSION__;
```

Comment:
no need for this one, you can recover the version from `this.config.grapesjs.version`

Reply:
I cant seem to get this to work, or find a reference for it in the code. 

```
TypeError: Cannot read properties of undefined (reading 'version')

      1153 |
      1154 |     // @ts-ignore
    > 1155 |     const version = this.config.grapesjs.version;
           |                                          ^
      1156 |     const sessionKey = `${sessionKeyPrefix}${version}`;
      1157 |
      1158 |     if (sessionStorage.getItem(sessionKey)) {
```